tgui-next/packages/tgui/interfaces/OreBox.js

```diff
@@ -0,0 +1,48 @@
+import { act } from '../byond';
+import { Fragment } from 'inferno';
+import { BlockQuote, Box, Table, Button, Section } from '../components';
+import { toTitleCase } from 'common/string';
+
+export const OreBox = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const { materials } = data;
+  return (
+    <Fragment>
+      <Section
+        title="Ore Box"
+      >
+        <BlockQuote mb={1}>
+          If you are wearing a mining stachel on your belt or in a pocket and dragging the box,
+          all ores will be placed in here.<br />
+          Gibtonite is not accepted.
+        </BlockQuote>
+        <Box>
+          <Box inline color="label" mr={1}>
+            Remove all ores from the box
+          </Box>
+          <Button
```

Comment:
You can take the <button> contents and put them in the <section> header (under line 14) and it'll put that in the same line as the section title. Give that a shot and see how it looks, I don't really think an explanation for what "Empty" does is necessary and this probably flows better.

Reply:
This is changed up to look better now.